wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/main.kt

```diff
@@ -0,0 +1,182 @@
+import com.squareup.wire.EnumAdapter
+import com.squareup.wire.FieldEncoding
+import com.squareup.wire.ProtoAdapter
+import com.squareup.wire.ProtoReader
+import com.squareup.wire.ProtoWriter
+import com.squareup.wire.WireEnum
+import com.squareup.wire.internal.Internal.missingRequiredFields
+import com.squareup.wire.kotlin.JavaPerson
+import com.squareup.wire.kotlin.Person2
+import com.squareup.wire.kotlin.UNKNOWN_FIELD
+import com.squareup.wire.kotlin.decodeMessage
+import com.squareup.wire.kotlin.getAlien
+import okio.ByteString
+
+data class Person(
+    val name: String,
```

Comment:
Should we keep `@WireField` annotations on the properties? To be fair no idea what they're used for

Reply:
I discussed with @swankjesse it is safe to delete them for now